AAT_raspberry/storage.py

```diff
@@ -0,0 +1,20 @@
+filename = 'store'
+
+# inserts val to end of store file
+def insert(val):
+  with open(filename, 'a') as f:
+    f.write(val + '\n')
+
+# gets all stored values
+def get():
+  with open(filename, 'r') as f:
+    return map(str.strip, f.readlines())
+
+# deletes first numLines number of values
+def delete(numLines):
+  lines = []
+  with open(filename, 'r') as f:
+    lines = f.readlines()
+
+  with open(filename, 'w') as f:
+    f.writelines(lines[numLines:])
```

Comment:
Don't we need to clear old content of the file before this write?
I think it is a good practice to close file after these operations.